Automattic/newspack-plugin
# feat: update reader registration block to use newspack-ui

### All Submissions:






### Changes proposed in this Pull Request:



The main goal of this PR is to update the Reader Registration block to use Newspack UI, aligning it with the rest of the RAS-ACC flow.

I've removed the styles (Stacked / Columns) as they're no longer relevant, and simplified the "Sign in" field to just a link/button (removing the need for "Already have an account?"), to further align with RAS-ACC.

On the Newspack UI end, I've made a few changes: I've replaced Inter with a new version and added Italic. I've enqueued the script into the editor to make sure the styles are available there.

![image](https://github.com/Automattic/newspack-plugin/assets/177929/53e81d87-f693-4721-8011-2b361b7f0eb9)

### How to test the changes in this Pull Request:

1. Add a Reader Registration block to a page.
2. Switch to this branch and make sure the block still looks fine.
3. Play with the different settings, etc...

### Other information:
